scripts/wget/1.19.2/script.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+MASON_NAME=wget
+MASON_VERSION=1.19.2
+MASON_LIB_FILE=bin/wget
+
+. ${MASON_DIR}/mason.sh
+
+function mason_load_source {
+    mason_download \
+        http://ftp.gnu.org/gnu/wget/wget-1.19.2.tar.gz \
+        07a689125eaf3b050cd62fcb98662eeddc4982db
+
+    mason_extract_tar_gz
+
+    export MASON_BUILD_PATH=${MASON_ROOT}/.build/${MASON_NAME}-${MASON_VERSION}
+}
+
+function mason_compile {
+    ./configure \
+        --prefix ${MASON_PREFIX}
+
+    PREFIX=${MASON_PREFIX} \
+    CXXFLAGS="${CXXFLAGS}" \
+    LDFLAGS="${LDFLAGS} -ldl -lpthread" make
+
+    PREFIX=${MASON_PREFIX} \
+    CXXFLAGS="${CXXFLAGS}" \
+    LDFLAGS="${LDFLAGS} -ldl -lpthread" make install
```

Comment:
Lines 23 -> 29 can likely be removed. The reason is that build systems that use `./configure` usually pick up these environment variables automatically, such that they do not need to be sent directly to `make`. A good example to follow for a `./configure` based build is harfbuzz: https://github.com/mapbox/mason/blob/master/scripts/harfbuzz/1.4.4-ft/script.sh#L36-L61. In harfbuzz the build variables are setup before `./configure`

Reply:
👍 Removed & Tested